servicenow/servicenow.js

```diff
@@ -0,0 +1,164 @@
+//Copyright 2020 Google LLC
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//    https://www.apache.org/licenses/LICENSE-2.0
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+
+(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {        
+
+
+        var writer = response.getStreamWriter();
```

Comment:
The intent of this file is not consistent with other source code under the repo, you may change intent to 2 spaces. 
The code format should be similar as: https://github.com/GoogleCloudPlatform/dialogflow-integrations/blob/master/spark/server.js

Reply:
re-formatted the code